atat_provisioning_wizard_api.yaml

```diff
@@ -333,7 +333,6 @@ paths:
           description: The given Portfolio Draft does not exist
         '404':
           description: Funding Step not found for this Portfolio Draft
-      x-amazon-apigateway-request-validator: "request-params-validator"
```

Comment:
We might want to talk with the team if we should keep `param-validator` in APIGW or not.

Reply:
Per slack we decided to include this in #381, so no need to include here.  We'll use the API GW validation to test for path parameter existence but do all other validation within lambdas.